### ISSUE TITLE
"accepts" not firing as event

### ISSUE THREAD

**Issue by jsisson** [kristianmandrup/vue2-dragula#17]
In your dragEffects example, within the [service.on](https://github.com/kristianmandrup/vue2-dragula-demo/blob/master/src/components/DragEffects.vue#L68) block you are registering the 'accepts' event, however vue2-dragula doesn't fire 'accepts' as an event.  I turned on full logging and 'accepts' never pops up.  In dragula 'accepts' is a configuration option.  

I was able to make it work thoughby adding it to the [createService](https://github.com/kristianmandrup/vue2-dragula-demo/blob/master/src/components/DragEffects.vue#L47) call;

```
...
let service = dragula.createService({
    name: 'effects',
    drake: {
        copy: true,
        accepts: (el, target, source, sibling) => {
            log('accepts: ', el, target)
            return true // target !== document.getElementById(left)
        },
    }
});
...
```

The library should either start firing 'accepts' as an event, or update the documentation.

**Comment by jredrejo** [comment]
Hi, same problem with moves event. It is not fired either

**Comment by kristianmandrup** [comment]
You are most welcome to extend the plugin with support for additional events, like accepts and move by enhancing `createService` internals etc. and making a PR. Then I will merge and make a new release with better docs as well.

Need some community support ;) too busy with many other projects...

**Comment by kristianmandrup** [comment]
Don't the [events](https://github.com/kristianmandrup/vue2-dragula#events) work?
Also see [$dragula](https://github.com/kristianmandrup/vue2-dragula#dragula) for direct access to dragula model internals.
For [drag effects/event](https://github.com/kristianmandrup/vue2-dragula#adding-drag-effects)

Not sure what else you would want. Note: I'm not a super dragula expert

**Comment by kristianmandrup** [comment]
I've now added the events you asked for:

```js
export class DragulaService {
// ...

    // Dragula events supported
    this.events = [
      'cancel',
      'cloned',
      'drag',
      'dragend',
      'drop',
      'out',
      'over',
      'remove',
      'shadow',
      'dropModel',
      'removeModel',
      'accepts',
      'moves',
      'invalid'
    ]
}
```

Updated the [docs](https://github.com/kristianmandrup/vue2-dragula#dragula-events-and-drag-effects) and added explicit support in [calcOpts](https://github.com/kristianmandrup/vue2-dragula/blob/master/src/service.js#L190) which can now be overriden if needed, by subclassing `DragulaService` and passing options to plugin to use your own service factory method.

Hope that helps. Sorry, haven't worked on this project for the past 12 months or so...

**Comment by kristianmandrup** [comment]
The latest changes are available on master as version 2.5.1. Let me know if it works out, then I will make a release with these additions.
